db/migrate/20230413112141_add_accredited_provider_id_to_provider.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAccreditedProviderIdToProvider < ActiveRecord::Migration[7.0]
+  def change
+    add_column :provider, :accredited_provider_id, :integer
```

Comment:
Do we need an index for this?

Reply:
yup fair enough, just wondered whether we'd ever search on it but probably not yet

Reply:
Cool lets stick with YAGNI until there is a plausible future for it :smile_cat: